### DISCUSSION TITLE
FFI Install?

### DISCUSSION THREAD

**Discussion by KatyaJansen** [Homebrew/discussions#2495] [Getting started] [answered]
I'm new to Brew. Can FFI be installed? I would normally install via GEM with `gem install ffi`.

Thanks

Katya

**Comment by SMillerDev** [comment, marked as answer]
The ffi library can be installed with `brew install libffi`. The ffi gem you can install the way you're used to.

**Reply by KatyaJansen** [reply]
Thanks for the fast reply! Would there be a scenario where I would need both? Is there a difference between the two?

**Reply by SMillerDev** [reply]
Libffi is a library on your system that the ffi gem can use, on macOS there is also a version included in the system so while you always need `gem install ffi` you might not need `brew install libffi` on macOS.

**Reply by KatyaJansen** [reply]
Great! Thanks
